convert2rhel/unit_tests/subscription_test.py

```diff
@@ -229,4 +232,19 @@ def test_parse_sub_date(self):
         sku = self.FakeSubscription()
         for i in sku.dates_formats:
             self.assertEqual(subscription.parse_sub_attrs(sku(i))["ends"], i)
-            self.assertEqual(len(subscription.logging.getLogger.critical_msgs),0)
+            self.assertEqual(len(subscription.logging.getLogger.critical_msgs), 0)
+
+
+    @unit_tests.mock(utils, "run_subprocess", RunSubprocessMocked())
+    def test_unregister_system(self):
```

Comment:
You can add a test that the logging happens for both the return code 0 and non-0.